### ISSUE TITLE
type inference using type coversion

### ISSUE THREAD

**Issue by shashank-g12** [SMAT-Lab/Scalpel#120]
Using type conversion operators such as int() or str() is not taken into  account when assigning type information to variables. For eg:
`import math

def main():
    a = int(input())
    b = int(input())
    bstr = str(b)
    num = a * 10 ** len(bstr) + b

    for i in range(1, 100100):
        if i * i == num:
            break

    if i > 100100:
        print("No")
    else:
        print("Yes")
main()
`
In this type infered for variable a and b should be int  and for bstr it should be string but type inference module shows 'Any'

**Comment by Jarvx** [comment]
Indeed, this is a bug! Thanks for reporting.  I will close this issue once this is fixed
